### PR TITLE
refactor(voting): Use default strategy to reduce voting request

### PR DESCRIPTION
**Pull request by chef-jojo**
- no need `deployedBlockNumber`
- use `cake` strategy https://github.com/snapshot-labs/snapshot-strategies/tree/master/src/strategies/cake when possible

### CODE REVIEW

File: src/views/Voting/helpers.ts [hunk 1]
```diff
@@ -80,9 +82,62 @@ export const sendSnapshotData = async (message: Message) => {
...
 }
 
+const votingPowerContractAddress = '0xc0FeBE244cE1ea66d27D23012B3D616432433F42'
+
+const votingPowerAbi = [
+  'function getCakeBalance(address _user) view returns (uint256)',
+  'function getCakeBnbLpBalance(address _user) view returns (uint256)',
+  'function getCakePoolBalance(address _user) view returns (uint256)',
+  'function getCakeVaultBalance(address _user) view returns (uint256)',
+  'function getIFOPoolBalancee(address _user) view returns (uint256)',
```
Comment (memoyil) [inline src/views/Voting/helpers.ts:92]: Typo

File: src/views/Voting/helpers.ts [hunk 2]
```diff
@@ -80,9 +82,62 @@ export const sendSnapshotData = async (message: Message) => {
...
+    [total],
+  ] = await multicallv2(votingPowerAbi, [...calls, ...poolCalls], {
+    blockTag: blockNumber,
+  })
+  return {
+    poolsBalance: formatEther(poolsBalance),
+    total: formatEther(total),
+    cakeBalance: formatEther(cakeBalance),
+    cakeVaultBalance: formatEther(cakeVaultBalance),
+    IFOPoolBalance: formatEther(ifoPoolBalance),
```
Comment (memoyil) [inline src/views/Voting/helpers.ts:136]: lowercase variable name is better for readability wdyt?

File: src/views/Voting/helpers.ts (hunk 1 above)
Comment (chef-jojo) [reply src/views/Voting/helpers.ts:92]: that's a typo on contract lol

Comment (memoyil) [reply src/views/Voting/helpers.ts:92]: :D

**Review by chefilip — APPROVED** [review]

**Pull request merged**